scripts/proposals/proposal_03_revoke_CM.js

```diff
@@ -16,7 +16,7 @@ async function main() {
     const CMAddress = parsedData.CM;
 
     // Obtaining proposal values
-    console.log("19. Revoking CM in the Timelock");
+    console.log("3. Revoking CM in the Timelock");
```

Comment:
none of this will be done since we are using the guard. Maybe it is less confusing if we remove this, no?

Reply:
Let it be in the proposals, it is a good practice to have something for all the contracts.